src/optimistic-oracle/optimistic-oracle.js

```diff
@@ -103,10 +88,10 @@ async function getPrice(contractAddress) {
 
   const price = response.data[tokenAddress][vsCurrency];
 
-  return price;
+  return parseFloat(price);
 }
 
-function provideHandleTransaction(erc20Contract) {
+function provideHandleTransaction(erc20MockContract = undefined) {
```

Comment:
this is implied and imo super super ugly. if you don't pass something for `erc20MockContract` it will be undefined. this is the default behaivor. very gross

Reply:
```
> function myFunc(myArg) {
... console.log(myArg);
... }
undefined
> myFunc();
undefined
undefined
> myFunc(1);
1
undefined
```

Reply:
Fixed